elfi/methods/model_selection.py

```diff
@@ -0,0 +1,56 @@
+"""This module contains methods for model comparison and selection."""
+
+import numpy as np
+
+
+def compare_models(sample_objs, model_priors=None):
+    """Find posterior probabilities for different models.
+
+    The algorithm requires elfi.Sample objects from prerun inference methods. For example the
+    output from elfi.Rejection.sample is valid. The portion of samples for each model in the top
+    discrepancies are adjusted by each models acceptance ratio and prior probability.
+
```

Comment:
The implementation of method is based solely on the discrepancies. An implicit requirement for the method is that the elfi models use the same discrepancy measure (along with the same set of summary statistics). Is there a way to neatly check this? Or would adding a warning about it be enough?

Reply:
Just under this line there's a statement: "The discrepancies must be comparable so that it is meaningful to sort them!"

It could be checked whether the `elfi.Discrepancy` nodes perform the same operation. However, I don't think that this is either a necessary or a sufficient condition for comparability, since the summaries play a role as well, and I suppose it's possible to come up with a non-identical distance function that yet are comparable.